pkgs/applications/networking/cluster/nomad/default.nix

```diff
@@ -1,8 +1,8 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGo112Package, fetchFromGitHub }:
 
-buildGoPackage rec {
+buildGo112Package rec {
```

Comment:
can you do:
```
    buildGoPackage = buildGo112Package;
```
in top-level/all-packages.nix please :), then revert the change in the package :)

Reply:
As a specific override for this package? Or?

Reply:
```
  nomad = callPackage ../applications/networking/cluster/nomad { 
    buildGoPackage = buildGo112Package;
  };
```

Reply:
ah - thanks!

Reply:
Done!